src/routers/TabRouter.js

```diff
@@ -32,6 +32,8 @@ export default (
   const initialRouteIndex = order.indexOf(initialRouteName);
   const backBehavior = config.backBehavior || 'initialRoute';
   const shouldBackNavigateToInitialRoute = backBehavior === 'initialRoute';
+  const shouldBackNavigateToLastActiveTab = backBehavior === 'previousRoute';
+  const backstack = [];
```

Comment:
ok I did an update where backstack is in NavigationState, I have them as indexes, let me know if that is fine or if route objects are needed.